ssas/groups.go

```diff
@@ -51,17 +51,28 @@ type GroupList struct {
 	Groups     []GroupSummary `json:"groups"`
 }
 
-func CreateGroup(ctx context.Context, gd GroupData, trackingID string) (Group, error) {
-	event := Event{Op: "CreateGroup", TrackingID: trackingID}
-	OperationStarted(event)
+type GroupData struct {
```

Comment:
no functional difference, moving the types to the top of the file for readability and per go's recommendation.